docs/model/introduction-to-model-architecture.md

```diff
@@ -1,21 +1,21 @@
-# Basics About Model Architecture
+# Introduction to Model Architecture
 
-In this article you will learn about [entities](#entity), [facades](#facade), [repositories](#repository) and their mutual relations.
+In this article you will learn about model, its dependencies, [entities](#entity), [facades](#facade), [repositories](#repository) and their mutual relations.
 
-## Basics about the model
+## Definition of a model
+Definition of model is adopted from [DDD](https://stackoverflow.com/questions/1222392/can-someone-explain-domain-driven-design-ddd-in-plain-english-please/1222488#1222488) definition of a model. Model is a system of abstractions that describes selected aspect of a domain.
 
-Model architecture of Shopsys Framework is inspired by Domain Driven Design (DDD). Let us define some terms first:
-- **Domain** in DDD is a sphere of knowledge or activity we build application logic around. The domain of Shopsys Framework is e-commerce.
-- **Domain model** is a system of abstractions that describes selected aspects of the domain.
-- **Domain logic** or **business logic** is the higher level rules for how objects of the domain model interact with one another.
+Domain is a sphere of knowledge or activity we build application logic around. The domain of Shopsys Framework is e-commerce. In Shopsys Framework we also use term domain for another use case, which is the domain url address and things related to them, you can read more about Domain in [Domain, Multidomain, Multilanguage](../introduction/domain-multidomain-multilanguage.md)
```

Comment:
The full copy of the paragraph seems a bit too much. It takes the focus away from domain model. Why did you use it instead of my suggestion? Do you think it's important to explain it fully here?